imap_processing/ultra/constants.py

```diff
@@ -67,3 +67,7 @@ class UltraConstants:
     ALPHA = 0.2  # deltaE/E
     ENERGY_START = 3.385  # energy start for the Ultra grids
     N_BINS = 23  # number of energy bins
+
+    # Constants for species determination
+    SPECIES_MIN = 50
+    SPECIES_MAX = 200
```

Comment:
Perhaps a more descriptive name or at least comment? These are min/max of CTOF to consider an event to be an ENA, right?